deploy/helm/values.yaml

```diff
@@ -10,7 +10,7 @@ fullnameOverride: ""
 imagePullSecrets:
 
 otel:
-  # e.g. otel-collector.dc-01.cloud.solarwinds.com:443
+  # e.g. otel.collector.na-01.cloud.solarwinds.com:443
```

Comment:
Can we add there link with all endpoints?

https://documentation.solarwinds.com/en/success_center/observability/content/system_requirements/endpoints.htm#Find

Reply:
changed